packages/grid/data-grid/src/tests/selection.DataGrid.test.tsx

```diff
@@ -235,7 +290,7 @@ describe('<DataGrid /> - Selection', () => {
       const row0 = getRow(0);
       expect(row0).to.have.class('Mui-selected');
 
-      setProps({ selectionModel: 1 });
+      setProps({ selectionModel: [1] });
```

Comment:
For any refactoring, changes in tests should receive high scrutiny. Do we need these changes in this PR? Why? Could we isolate them?

Reply:
I think it was to have the test pass because Damien did not have the time to handle properly the non-array  version.
But it's now implemented so I can revert this change.